app/view_models/auction_show_view_model.rb

```diff
@@ -222,23 +222,11 @@ def status_presenter
     @_status_presenter ||= StatusPresenterFactory.new(auction).create
   end
 
-  def for_small_business?
-    AuctionThreshold.new(auction).small_business?
-  end
-
   def current_user_can_bid?
-    if !available? || auction.type == 'single_bid' && user_is_bidder?
-      false
-    elsif current_user.is_a?(Guest)
-      true
-    elsif !current_user.sam_accepted?
-      false
-    elsif for_small_business? && current_user.small_business?
-      true
-    elsif for_small_business? && !current_user.small_business?
-      false
-    else # not for small business
+    if current_user.is_a?(Guest)
```

Comment:
I wonder if this logic should be swept into the Rules class? Either that or rename `current_user_can_bid?` to `show_bid_button?` maybe?